zfit/models/physics.py

```diff
@@ -49,14 +49,16 @@ def crystalball_integral(limits, params, model):
     alpha = params['alpha']
     n = params['n']
 
-    (lower,), (upper,) = limits.limits
-    lower = lower[0]  # obs number 0
-    upper = upper[0]
+    lower, upper = limits._rect_limits_tf
+    lower = lower
+    upper = upper
 
-    return crystalball_integral_func(mu, sigma, alpha, n, lower, upper)
+    integral = crystalball_integral_func(mu, sigma, alpha, n, lower, upper)
+    return integral
 
 
-@z.function_tf
+# @z.function_tf_input
+# @tf.function  # BUG? TODO: problem with tf.function and input signature
```

Comment:
delete

Reply:
Maybe I've added the comment afterwards? Should be there since it's an open bug.